concepts/classes/about.md

```diff
@@ -0,0 +1,260 @@
+# About
+
+JavaScript includes the capabilities for object-oriented programming ([OOP][wiki-oop]).
+In OOP, you want to create objects (_instances_) from "templates" (_classes_) so that they include certain data and functionality.
+The data properties are called _fields_ in the OOP context, the function properties are called _methods_.
+
+JavaScript did not have classes at all before they were added to the language specification in 2015.
```

Comment:
```suggestion
JavaScript did not have classes at all before they were added to the language specification in 2015, but allowed for object-oriented programming using prototype-based inheritance. 
```

Perhaps we clarify why this is mentioned at all.

Reply:
Good point!